airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -2109,3 +2128,265 @@ def execute(self, context: Any):
+            validation results?  Defaults to False because there's extra setup needed to serve HTML data docs stored in
+            GCS.  When set to False, only a GCS path to the results are included in the email.
+            Execute the following steps if you want a clickable link for the data doc to be included in the
+            email:
+            1) Set up a GAE app to serve the data docs by following the instructions here:
+            https://docs.greatexpectations.io/en/latest/guides/how_to_guides/configuring_data_docs/how_to_host_and_share_data_docs_on_gcs.html
+            2) Create an Airflow variable called 'great_expectations_datadocs_domain' with a value set to the domain
+               google creates in step 1 to serve the data docs (e.g. 'ge-data-docs-dot-my-gcp-project.ue.r.appspot.com').
+               This operator will look for it if include_datadocs_link_in_email=True
+        :type include_datadocs_link_in_email: boolean
+        :param email_to:  Email address to receive any alerts when expectations are not met.
+        :type email_to: str
+        :param fail_if_expectations_not_met: Fail the Airflow task if expectations are not met?  Defaults to True.
+        :type fail_if_expectations_not_met: boolean
+    """
+
+    @apply_defaults
+    def __init__(self, gcp_project, expectations_file_name, gcs_bucket, gcs_expectations_prefix,
+                 gcs_validations_prefix, gcs_datadocs_prefix, validation_type, validation_type_input,
+                 bq_dataset_name, email_to, send_alert_email=True, include_datadocs_link_in_email=False,
+                 fail_if_expectations_not_met=True,
+                 bigquery_conn_id='bigquery_default',
+                 *args, **kwargs):
+        self.expectations_file_name = expectations_file_name
+        if validation_type.upper() not in GreatExpectationsValidations:
+            raise AirflowException("argument 'validation_type' must be one of %r." % great_expectations_valid_type)
+        self.validation_type = validation_type
+        self.validation_type_input = validation_type_input
+        self.bigquery_conn_id = bigquery_conn_id
+        self.bq_dataset_name = bq_dataset_name
+        self.gcp_project = gcp_project
+        self.gcs_bucket = gcs_bucket
+        self.gcs_expectations_prefix = gcs_expectations_prefix
+        self.gcs_validations_prefix = gcs_validations_prefix
+        self.gcs_datadocs_prefix = gcs_datadocs_prefix
+        self.email_to = email_to
+        self.send_alert_email = send_alert_email
+        self.include_datadocs_link_in_email = include_datadocs_link_in_email
+        self.fail_if_expectations_not_met = fail_if_expectations_not_met
+
+        super(GreatExpectationsBigQueryOperator, self).__init__(*args, **kwargs)
+
+    def execute(self, context):
+        conn = BaseHook.get_connection(self.bigquery_conn_id)
+        connection_json = json.loads(conn.extra)
```

Comment:
There's a method for that:
```suggestion
        connection_json = conn.extra_dejson
```